Find a functionally equivalent code:android.view.ViewGroup.requestChildFocus:COMMENT
Method Modifier: public      
Comment:/**
 * {@inheritDoc}
 */

@@ -6,16 +6,16 @@
         return;
     }
     // Unfocus us, if necessary
-    super.unFocus();
+    super.unFocus(focused);
     // We had a previous notion of who had focus. Clear it.
     if (mFocused != child) {
         if (mFocused != null) {
-            mFocused.unFocus();
+            mFocused.unFocus(focused);
         }
         mFocused = child;
     }

